package.json

```diff
@@ -3,27 +3,48 @@
   "version": "2.2.0",
   "description": "Use LevelUP to cache remote data",
   "main": "index.js",
+  "engines": {
+    "node": ">=6.0.0"
+  },
   "directories": {
     "test": "test"
   },
   "devDependencies": {
     "asynquence": "0.8.2",
+    "babel-preset-es2015": "6.14.0",
```

Comment:
Yeah - my goal is to support two use cases: node via node 6, and ES5-compatible browsers via browserify.  Babel is needed to support the latter.